e2e/linter/src/linter.test.ts

```diff
@@ -572,10 +569,61 @@ describe('Linter', () => {
       ]);
       expect(e2eEslint.overrides[0].extends).toBeUndefined();
       expect(e2eEslint.overrides[1].extends).toBeUndefined();
-      expect(libEslint.overrides[0].extends).toEqual([
-        'plugin:@nrwl/nx/angular',
-        'plugin:@angular-eslint/template/process-inline-templates',
+      expect(libEslint.overrides[0].extends).toBeUndefined();
+    });
+
+    it('(Node standalone) should set root project config to app and e2e app and migrate when another lib is added', () => {
+      const myapp = uniq('myapp');
+      const mylib = uniq('mylib');
+
+      runCLI(
+        `generate @nrwl/node:app ${myapp} --rootProject=true --no-interactive`
+      );
+
+      let rootEslint = readJson('.eslintrc.json');
+      let e2eEslint = readJson('e2e/.eslintrc.json');
+      expect(() => checkFilesExist(`.eslintrc.base.json`)).toThrow();
+
+      // should directly refer to nx plugin
+      expect(rootEslint.plugins).toEqual(['@nrwl/nx']);
+      expect(e2eEslint.plugins).toEqual(['@nrwl/nx']);
+      // should only extend framework plugin
+      expect(e2eEslint.extends).toEqual([]);
+      // should have plugin extends
+      expect(rootEslint.overrides[0].extends).toEqual([
+        'plugin:@nrwl/nx/typescript',
+      ]);
+      expect(rootEslint.overrides[1].extends).toEqual([
+        'plugin:@nrwl/nx/javascript',
+      ]);
+      expect(e2eEslint.overrides[0].extends).toEqual([
+        'plugin:@nrwl/nx/typescript',
+      ]);
+      expect(e2eEslint.overrides[1].extends).toEqual([
+        'plugin:@nrwl/nx/javascript',
```

Comment:
nitpick: personally this doesn't read as an e2e test and instead is more unit test like. 
instead we should just run the lint target and assert that it works instead of checking properties of generated files (where a unit test is more likely to already cover this). since this test is now dependent on the order of the file contents instead of how the user interacts with Nx like a true e2e test should be.

I'd also recommend using expect.objectContaining and expect.arrayContaining instead of asserting each index of the override otherwise this test will have to be updated every time the array order changes which is a bit too much 'context' that an e2e test should have IMO.

Just a though.

Reply:
The reason we are checking the expected content, if that there is a scenario where `nx lint ...` would pass but running `eslint ...` would fail.

But you are correct about running `lint` command to verify things are working and using potentially smarter checks.

Reply:
I improved the tests, unfortunately, this doubled the duration of the test suite.